assets/src/edit-story/icons/page_layouts.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M1.5 0C0.671573 0 0 0.671573 0 1.5V7.5C0 8.32843 0.671573 9 1.5 9H5.5C6.32843 9 7 8.32843 7 7.5V1.5C7 0.671573 6.32843 0 5.5 0H1.5ZM1 1.5C1 1.22386 1.22386 1 1.5 1H5.5C5.77614 1 6 1.22386 6 1.5V7.5C6 7.77614 5.77614 8 5.5 8H1.5C1.22386 8 1 7.77614 1 7.5V1.5ZM1.5 11C0.671573 11 0 11.6716 0 12.5V14.5C0 15.3284 0.671573 16 1.5 16H5.5C6.32843 16 7 15.3284 7 14.5V12.5C7 11.6716 6.32843 11 5.5 11H1.5ZM1 12.5C1 12.2239 1.22386 12 1.5 12H5.5C5.77614 12 6 12.2239 6 12.5V14.5C6 14.7761 5.77614 15 5.5 15H1.5C1.22386 15 1 14.7761 1 14.5V12.5ZM9 1.5C9 0.671573 9.67157 0 10.5 0H14.5C15.3284 0 16 0.671573 16 1.5V4.5C16 5.32843 15.3284 6 14.5 6H10.5C9.67157 6 9 5.32843 9 4.5V1.5ZM10.5 1C10.2239 1 10 1.22386 10 1.5V4.5C10 4.77614 10.2239 5 10.5 5H14.5C14.7761 5 15 4.77614 15 4.5V1.5C15 1.22386 14.7761 1 14.5 1H10.5ZM10.5 8C9.67157 8 9 8.67157 9 9.5V14.5C9 15.3284 9.67157 16 10.5 16H14.5C15.3284 16 16 15.3284 16 14.5V9.5C16 8.67157 15.3284 8 14.5 8H10.5ZM10 9.5C10 9.22386 10.2239 9 10.5 9H14.5C14.7761 9 15 9.22386 15 9.5V14.5C15 14.7761 14.7761 15 14.5 15H10.5C10.2239 15 10 14.7761 10 14.5V9.5Z" fill="#E4E5E6"/>
+</svg>
```

Comment:
If you want, this is also in the new design system icons. Em referenced some alert icons in her prepubish checklist. this one would be `const { Layout } from '../../blah/.../design-system/icons/editor/'`

Reply:
Is that cool to do so instead of putting it in `edit-story`?

Reply:
i think so! There's no added dependencies and it's helping avoid duplication as new icons are added.